src/main/java/racingcar/service/CarRacing.java

```diff
@@ -0,0 +1,66 @@
+package racingcar.service;
+
+import racingcar.model.Car;
+import java.util.*;
+
+public class CarRacing {
+
+    private List<Car> carsStatus;
+
+    public CarRacing() {}
+
+    public CarRacing(int numberOfCars) {
+        carRaceReady(numberOfCars);
+    }
+
+    private void carRaceReady(int numberOfCars) {
+        makeCarsStatusList(numberOfCars);
+    }
```

Comment:
```suggestion
    private void carRaceReady(int numberOfCars) {
        List<Car> cars = new ArrayList<Car>();
        for (int i = 0; i < numberOfCars; i++) {
            cars.add(new Car());
        }
        this.cars = cars;
    }
```

makeCarsStatusList 호출 없이 내부 로직을 그대로 carRaceReady에 바로 작성해도 되지 않을까요?

만약 테스트를 위함이라면
메소드를 노출시키는 대신에 CarRacing 생성자를 사용해서 데이터를 만들어 볼 수 있을 것 같아요
```java
new CarRacing(3); // 자동차 3대 준비
```

Reply:
아래와 같은 생성자도 추가된다면, 더욱 명확해지고 테스트 편의성도 높아질 것 같습니다.
```java
new CarRacing(List.of(new Car(), new Car(), new Car())); // 자동차 3대 준비
```